src/main/java/pages/ComparingPage.java

```diff
@@ -41,20 +41,13 @@ public static String getComparingLink() {
         return comparingLink;
     }
 
-    public int amountOfComparingProducts() {
+    public String amountOfComparingProducts() {
         wait.tillElementsPresent(productsOnComparing);
-        return productsOnComparing.size();
+        return String.valueOf(productsOnComparing.size());
```

Comment:
Method should return 'amount', but you want to return string? Why do we need this?